content/_ext/pipeline.rb

```diff
@@ -22,7 +22,13 @@
                                                 '/rss.xml',
                                                 :feed_title => 'Jenkins Blog',
                                                 :template => '_ext/atom.xml.haml',
-                                                :num_entries => 20)
+                                                :num_entries => 50)
+
+  extension Awestruct::Extensions::Atomizer.new(:posts,
+                                                '/full-rss.xml',
+                                                :feed_title => 'Jenkins Blog',
+                                                :template => '_ext/atom.xml.haml',
+                                                :num_entries => 4096)
```

Comment:
We should create more blogposts to cause an overflow here! 😈

Reply:
:+1: